src/shared/ui/subtitle/subtitle.module.scss

```diff
@@ -0,0 +1,40 @@
+.subtitle {
+  margin: 0;
+  font-weight: $font-weight-medium;
+  line-height: 28px;
+  letter-spacing: 0;
```

Comment:
Why do we need `letter-spacing: 0;` here? as I can see, this does not affect the appearance of the title.